src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,28 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
```

Comment:
Implementing Cloneable and providing a clone method is not a common practice in entity classes. Entity instances should be unique and managed by the persistence context, cloning them can lead to unexpected behavior and issues with the persistence context. Consider removing the clone method and the Cloneable interface.